src/bootstrap/lib.rs

```diff
@@ -113,7 +113,7 @@
 //! More documentation can be found in each respective module below, and you can
 //! also check out the `src/bootstrap/README.md` file for more information.
 
-#![deny(warnings)]
```

Comment:
Shouldn't this be kept ?

Reply:
I wasn't getting any warnings when I ran my test build without it, so I removed it; was I too quick to do so? I can revert this bit it if I have missed some unpleasant consequence.

Reply:
Ah, you're right - I actually misread it as `#![allow(warnings)]` 🙈. I'll revert this bit.